webpack.config.js

```diff
@@ -50,6 +50,11 @@ if (DEV) {
     }
 }
 
+// Track the build time so that we can bust any caches that may have incorrectly cached remote_entry.js from before we
+// started setting Cache-Control: no-cache for that file on the server. This can be removed in 2024 after those cached
```

Comment:
Should we create a ticket?

Reply:
I don't think we need to. It's not particularly critical to do it since we're not caching anything either way, and we've got enough stuff in the backlog that it'll probably get forgotten about

Reply:
Ok, let's see if I remember this after a year :joy: